dataobj/environment.h

```diff
@@ -313,6 +314,9 @@ class env_t
 	/// Hide buildings and trees within range of mouse cursor
 	static uint16 cursor_hide_range;
 
+	static bool highlight_city;
+	static stadt_t *highlighted_city;
```

Comment:
Is `highlight_city` always false when `highlighted_city` is NULL? If so, only `highlighted_city` is enough.